doc/source/ray-core/patterns/map-reduce.rst

```diff
@@ -0,0 +1,33 @@
+Pattern: Using tasks to achieve MapReduce
+=========================================
+
+In this pattern, Ray remote tasks can be used to implement the MapReduce paradigm.
+For the ``map`` stage, we can submit multiple tasks to process data in a distributed and parallel fashion.
+For the ``reduce`` stage, we use :ref:`ray.get() <ray-get-ref>` (or implicitly via task arguments) to fetch the results of each of these tasks and aggregate on them.
```

Comment:
Remove ray.get() part since this is not recommended and it's also not used in the examples.